drivers/mssql/mssql.go

```diff
@@ -467,9 +536,17 @@ func (m *Mssql) Info() (*schema.Driver, error) {
 		return nil, err
 	}
 
+	dct := dict.New()
+	dct.Merge(map[string]string{
+		"Subroutines": "Stored procedures and functions",
```

Comment:
I am wondering if `Subroutines` is an appropriate name for the section.

Reply:
good point. Should I rename it to `ProceduresAndUDF` or something like that?